docs/cli-lib/api/getAndLoadConfigIfNeeded.md

```diff
@@ -0,0 +1,35 @@
+## `getAndLoadConfigIfNeeded(options)`
```

Comment:
You can view the formatted version [here](https://github.com/HubSpot/hubspot-cli/blob/23fc2afeca0ff8f2dabe8a32761c950537914494/docs/cli-lib/api/getAndLoadConfigIfNeeded.md)